apps/rpc/src/modules/event/event-repository.ts

```diff
@@ -56,13 +56,13 @@ export function getEventRepository(): EventRepository {
   return {
     async create(handle, data) {
       const row = await handle.event.create({ data })
-      const event = await this.findById(handle, row.id)
+      const event = await this.findById(handle, row.id, { includeDeleted: true })
```

Comment:
Shouldn't this be false? Under what circumstances would an event be deleted at the same time of creation?